### ISSUE TITLE
How to input my Azure speech API Key?

### ISSUE THREAD

**Issue by piwawa** [rany2/edge-tts#248]
The free quota is not enough for me, I want to maintain long-term use by calling my own Azure API. Do we have any solution?

**Comment by piwawa** [comment]
> There are no plans to support the official Azure API.

What is the daily limit? Is there any way to increase the number of uses?

**Comment by piwawa** [comment]
Is this calling your own API? Or is there any official free interface with limit?

**Comment by rany2** [comment]
No, it is using the API used by Microsoft Edge browser (same keys, etc.)

**Comment by piwawa** [comment]
> No, it is using the API used by Microsoft Edge browser (same keys, etc.)

Thanks for your answer.
